core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -107,7 +116,7 @@ export class ContributionsAndReview
   activeDropdownTabChoice: string;
   reviewTabs: TabDetails[] = [];
   contributionTabs: TabDetails[] = [];
-  tabNameToOpportunityFetchFunction: unknown;
+  tabNameToOpportunityFetchFunction: object;
```

Comment:
Can you use more specific type here?

Reply:
I will handle this while strict typing this file. This file is not yet fully strict typed.

Reply:
thanks @JeeveshGarg

Reply:
updated